css-layout-classes/style.css

```diff
@@ -0,0 +1,44 @@
+* {
+  font-family: sans-serif;
+}
+
+.container {
+  display: flex;
```

Comment:
The `container` class should not be a flexbox, it should only have `max-width` and `margin`